protocol-units/settlement/mcr/contracts/upgrader/README.md

```diff
@@ -0,0 +1,45 @@
+# Upgrader
+
+The Upgrader is a script designed to automate the upgrade process for settlement protocol units, streamlining the execution of several key steps required for contract upgrades. Below is a breakdown of its main components and the steps it performs.
+
+## Process Overview
```

Comment:
I would break out the overall high-level for an upgrade at the top. Something like...

Upgrades implementation contract with `upgrade.sh` -> Proposes upgrade to proxy contract with `propose.sh` -> Accepts proposal to upgrade proxy contract with `accept.sh`.